### PR TITLE
Refactors VoG into a proc/spell

### PR DESCRIPTION
**Pull request by XDTM**
Current VoG does not have any gameplay changes, but the voice_of_god proc can now be used by non-organs.
Added a spell that uses the Voice of God proc for badmins or future coders to use; the main benefit of this is that it can be given to non-carbons, allowing GODLY IAN. It does not activate on .x, however.

Cooldown modifier, power and speech spans are vareditable; i added a child spell as an example with the clown span, 0.1 power and 0.5 cooldown.

Requested by @coiax.

### CODE REVIEW

File: code/modules/spells/spell_types/voice_of_god.dm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+/obj/effect/proc_holder/spell/voice_of_god/can_cast(mob/user = usr)
+	if(!user.can_speak())
+		user << "<span class='warning'>You are unable to speak!</span>"
+		return FALSE
+	return TRUE
+
+/obj/effect/proc_holder/spell/voice_of_god/choose_targets(mob/user = usr)
+	perform(user=user)
+/obj/effect/proc_holder/spell/voice_of_god/perform(list/targets, recharge = 1, mob/user = usr)
```
Comment (optimumtact) [inline code/modules/spells/spell_types/voice_of_god.dm]: ugh, what is that user=usr, do spells really not get user passed through to them as a variable?

File: code/modules/spells/spell_types/voice_of_god.dm [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	command = input(user, "Speak with the Voice of God", "Command")
```
Comment (optimumtact) [inline code/modules/spells/spell_types/voice_of_god.dm]: this is a sleep so you need to check !qdeleted(src) and !qdeleted(user)

**Review by optimumtact — DISMISSED** [review]
Please add the qdel checks, and answer about the user

File: code/modules/spells/spell_types/voice_of_god.dm (hunk 1 above)
Comment (XDTM) [inline code/modules/spells/spell_types/voice_of_god.dm]: i've been basing it off other spells, so i kept the parameters

File: code/modules/spells/spell_types/voice_of_god.dm (hunk 2 above)
Comment (XDTM) [inline code/modules/spells/spell_types/voice_of_god.dm]: right, i'll fix it

**Comment by XDTM** [comment]
@optimumtact Changes applied as requested.

**Pull request merged**